examples/platform/silabs/efr32/wf200/host_if.cpp

```diff
@@ -1266,8 +1219,8 @@ void wfx_cancel_scan(void)
     if (scan_ssid)
     {
         vPortFree(scan_ssid);
-        scan_ssid = (char *) 0;
+        scan_ssid = NULL;
```

Comment:
you should use `nullptr`. it is standard for cpp. this comment applies to most places where you use NULL